kiwi/builder/container.py

```diff
@@ -80,13 +103,30 @@ def create(self):
 
         * image="docker"
         """
-        log.info(
-            'Setting up %s container', self.requested_container_type
-        )
-        container_setup = ContainerSetup(
-            self.requested_container_type, self.root_dir, self.container_config
-        )
-        container_setup.setup()
+        if not self.base_image:
+            log.info(
+                'Setting up %s container', self.requested_container_type
+            )
+            container_setup = ContainerSetup(
+                self.requested_container_type, self.root_dir,
+                self.container_config
+            )
+            container_setup.setup()
+        else:
+            try:
+                checksum = Checksum(self.base_image)
+                with open(self.base_image_md5) as md5_file:
+                    base_image_md5 = md5_file.read()
```

Comment:
I am Morgan Stanley. Or Nomura. So I want my checksums in SHA256. Or CosmicRay13. Or Nirvana777.